presto-main/src/main/java/io/prestosql/server/PrestoSystemRequirements.java

```diff
@@ -96,15 +96,15 @@ private static void verifyJavaVersion()
         }
 
         JavaVersion version = JavaVersion.parse(javaVersion);
-        if (version.getMajor() == 8 && version.getUpdate().isPresent() && version.getUpdate().getAsInt() >= 151) {
+        if (version.getMajor() == 8 && version.getUpdate().isPresent() && version.getUpdate().getAsInt() >= 162) {
```

Comment:
in https://github.com/prestosql/presto/pull/778#discussion_r284410987 @dain mentioned u161

Reply:
I found a different blog which mentioned 162. I'll change this to 161.